packages/adapters/src/adapters/Cornerstone/ContourRoi.js

```diff
@@ -0,0 +1,82 @@
+import { utilities } from "dcmjs";
```

Comment:
@sedghi - should this file just be deleted for now, until we figure out how to serialize all the contour annotations to SR?

Reply:
I thought this is working, you mean you haven't tested it? Yes please delete if you think it is hard to test for now